echoregions/convert/api.py

```diff
@@ -35,7 +35,6 @@ def read_evr(
     """
     return Regions2D(
         input_file=str(filepath),
-        parse=True,
         offset=offset,
```

Comment:
I think all these references to offset can be removed.

Reply:
Hi, @valentina-s Could you retake a look at this PR and approve it if I've sufficiently removed offset?